### PR TITLE
awsconfig/v7: encrypt etcd EBS volume

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/2599

### CODE REVIEW

File: service/awsconfig/v7/version_bundle.go [hunk 1]
```diff
@@ -49,7 +54,7 @@ func VersionBundle() versionbundle.Bundle {
 		Deprecated:   false,
 		Name:         "aws-operator",
 		Time:         time.Date(2018, time.February, 27, 14, 16, 0, 0, time.UTC),
-		Version:      "2.1.3",
+		Version:      "3.0.0",
```
Comment (kopiczko) [inline service/awsconfig/v7/version_bundle.go]: This is a backward incompatible change. We can not upgrade existing clusters.

File: service/awsconfig/v7/templates/cloudformation/guest/instance.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ const Instance = `{{define "instance"}}
     DependsOn:
     - MasterInstance
     Properties:
+      Encrypted: true
```
Comment (rossf7) [inline service/awsconfig/v7/templates/cloudformation/guest/instance.go]: For my interest how is the KMS key chosen if its not using `KmsKeyId`?

File: service/awsconfig/v7/version_bundle.go (hunk 1 above)
Comment (rossf7) [reply service/awsconfig/v7/version_bundle.go]: Ack. Major version bump makes sense in this case.

**Comment by kopiczko** [comment]
Still fishing for LGTM

**Comment by rossf7** [comment]
I asked a question on how the KMS key was chosen. I want to check that before approving. Otherwise this looks fine.

File: service/awsconfig/v7/templates/cloudformation/guest/instance.go (hunk 2 above)
Comment (kopiczko) [reply service/awsconfig/v7/templates/cloudformation/guest/instance.go]: It uses aws/ebs key for region. I checked that in the UI.

**Comment by kopiczko** [comment]
Sorry, I forgot to answer.

**Comment by xh3b4sd** [comment]
I think we do not have enough data points to qualify a single commit for this. Would be nice to get metrics into the integration tests on our side. Then we can maybe annotate with commits or something and can reason about this more properly.

**Comment by kopiczko** [comment]
Re performance I don't know, but last time I was testing on ginger I was waiting super long for DNS to propagate.

**Comment by fgimenez** [comment]
I've been reviewing execution times and it doesn't look directly related to these changes, we have moved from execution times most of the times around 28min one week ago https://circleci.com/gh/giantswarm/workflows/aws-operator?page=5 to most of the times over 32min in the last two days https://circleci.com/gh/giantswarm/workflows/aws-operator and https://circleci.com/gh/giantswarm/workflows/aws-operator?page=2, we had these worse times before this branch was merged.

**Pull request merged**